src/deadline/job_attachments/api/manifest.py

```diff
@@ -247,3 +260,182 @@ def process_output(status: FileStatus, path: str, output_diff: ManifestDiff):
             process_output(fast_diff_item[1], fast_diff_item[0], output)
 
     return output
+
+
+def _manifest_upload(
+    manifest_file: str,
+    s3_bucket_name: str,
+    manifest_path: str,
+    s3_metadata: Dict[str, Any],
+    boto_session: boto3.Session,
+    logger: ClickLogger = ClickLogger(False),
+):
+    """
+    BETA API - This API is still evolving but will be made public in the near future.
+    API to upload a job attachment manifest to the Content Address Storage. Manifests will be
+    uploaded to s3://{s3_bucket_name}/Manifest/{manifest_path} as per the Deadline CAS folder structure.
+    manifest_file: File Path to the manifest file for upload.
+    s3_bucket_name: S3 bucket name.
+    manifest_path: S3 path for the manifest.
+    s3_metadata: Additional S3 file metadata tagged on upload.
+    boto_session: Boto3 session.
+    logger: Click Logger instance to print to CLI as test or JSON.
+    """
+    # Always upload the manifest file to case root /Manifest with the original file name.
+    manifest_path = manifest_path + "/Manifests/" + Path(manifest_file).name
```

Comment:
Should we reuse the same variable for `Manifests` folder here as well?

```suggestion
    manifest_path = "/".join([manifest_path, S3_MANIFEST_FOLDER_NAME, Path(manifest_file).name])
```

Reply:
Great idea